examples/pylintrc

```diff
@@ -1,5 +1,22 @@
 [MASTER]
 
+# Analyse import fallback blocks. This can be used to support both Python 2 and
+# 3 compatible code, which means that the block might have code that exists
+# only in one or another interpreter, leading to false positives when analysed.
+analyse-fallback-blocks=no
+
+# Load and enable all available extensions. Use --list-extensions to see a list
+# all available extensions.
+enable-all-extensions=None
```

Comment:
There are a lot of suspicious "None" value in this file that look like they should be empty strings.

Reply:
I was thinking the same. Is ``--generate-rcfile`` broken?

Reply:
~~I don't think it's broken but~~, yes definitely broken:
````
Traceback (most recent call last):
  File "/home/pierre/pylint/venv/bin/pylint", line 11, in <module>
    load_entry_point('pylint', 'console_scripts', 'pylint')()
  File "/home/pierre/pylint/pylint/__init__.py", line 25, in run_pylint
    PylintRun(argv or sys.argv[1:])
  File "/home/pierre/pylint/pylint/lint/run.py", line 135, in __init__
    args = _config_initialization(
  File "/home/pierre/pylint/pylint/config/config_initialization.py", line 56, in _config_initialization
    linter._parse_configuration_file(config_args)
  File "/home/pierre/pylint/pylint/config/arguments_manager.py", line 242, in _parse_configuration_file
    self.config, parsed_args = self._arg_parser.parse_known_args(
  File "/usr/lib/python3.8/argparse.py", line 1812, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/lib/python3.8/argparse.py", line 2018, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/usr/lib/python3.8/argparse.py", line 1958, in consume_optional
    take_action(action, args, option_string)
  File "/usr/lib/python3.8/argparse.py", line 1886, in take_action
    action(self, namespace, argument_values, option_string)
  File "/home/pierre/pylint/pylint/config/callback_actions.py", line 422, in __call__
    self.linter._load_reporters(values[0])
  File "/home/pierre/pylint/pylint/lint/pylinter.py", line 373, in _load_reporters
    reporter = self._load_reporter_by_name(reporter_name)
  File "/home/pierre/pylint/pylint/lint/pylinter.py", line 403, in _load_reporter_by_name
    raise exceptions.InvalidReporterError(name) from e
pylint.exceptions.InvalidReporterError: none
```

Reply:
Guess we found another `beta-blocker` 😓

Reply:
I'm glad we thought about it a little and opened a new MR instead of rushing the release 😄